packages/apps/demo/src/Demo.tsx

```diff
@@ -12,13 +12,26 @@ import { useState } from 'react';
 import MapGl, {
   AttributionControl,
   FullscreenControl,
+  Marker,
   NavigationControl,
 } from 'react-map-gl/maplibre';
+import { useSearchParams } from 'react-router-dom';
+import './Demo.css';
 import { createListProps, Legend } from './Legend';
 import { MapSelectAndSearch } from './MapSelectAndSearch';
+import { ShareControl } from './ShareControl';
 import { toStateCodes } from './state-codes';
 
 const Demo = () => {
+  const [searchParams] = useSearchParams();
+  const markerPos =
+    searchParams.has('mlat') && searchParams.has('mlon')
+      ? {
+          lat: Number(searchParams.get('mlat')),
+          lon: Number(searchParams.get('mlon')),
```

Comment:
Oof... looks like I need to do some validation/cleanup. When manually entering a URL with junk params:

<img width="372" alt="image" src="https://github.com/truckermudgeon/maps/assets/121829201/3a80ebfe-8683-47b4-80cd-e7d0e4b30ef3">

Reply:
https://github.com/truckermudgeon/maps/pull/2/commits/43046809d61e8c9fcdf17654530ad0c6218bdabd